### ISSUE TITLE
Different return values of GetOccurrences compared with DDay.iCal

### ISSUE THREAD

**Issue by gomdolkim** [ical-org/ical.net#158]
As I tested the following code, occurrences has IPeriod that StartTime is 2016-01-01 and Endtime is 2016-01-02.
Compared with DDay.iCal, that was 2016-01-01 ~ 2016-01-05.

I changed the code and it works same as DDay.iCal. 

```
public class EventEvaluator : RecurringEvaluator
{
            // foreach (var period in Periods.Where(period => period.EndTime == null)) 
            foreach (var period in Periods) 
            {
                period.Duration = Event.Duration;
                period.EndTime = period.Duration == null ? period.StartTime : period.StartTime.Add(Event.Duration);
            }
}
```

```
           var vEvent = new Event
            {
                DtStart = new CalDateTime(DateTime.Parse("2016-01-01")),
                DtEnd = new CalDateTime(DateTime.Parse("2016-01-05")),
            };

            var vEvent2 = new Event
            {
                DtStart = new CalDateTime(DateTime.Parse("2016-03-01")),
                DtEnd = new CalDateTime(DateTime.Parse("2016-03-05")),
            };

            var calendar = new Calendar();
            calendar.Events.Add(vEvent);
            calendar.Events.Add(vEvent2);

            var searchStart = DateTime.Parse("2015-12-29");
            var searchEnd = DateTime.Parse("2017-02-10");
            var occurrences = calendar.GetOccurrences(searchStart, searchEnd);
```

Is there any purpose to return values differently? or kind of bug?

**Comment by gomdolkim** [comment]
Could you help this issue?

**Comment by rianjs** [comment]
Fix published in Nuget package 2.2.15
